packages/sdk/src/libs/tables/blocks/js-map.ts

```diff
@@ -1,13 +1,9 @@
-import type { Data } from ".";
-import { JsMap as GenericJsMap } from "../generic";
+import { Data, makeId } from "./utils";
+import { JsMap } from "../generic";
 
 // personally dont like to have this as a named export, but cannot export both function and type as default
-export const JsMap = (type = "Block") => {
-  function makeId(data: Data) {
-    return data.number;
-  }
-
-  const table = GenericJsMap<number, Data>(type, makeId);
+export const Table = (type = "Block") => {
```

Comment:
^would classes solve this exporting issue?

Reply:
yes it would. i wanted to do a test with classes, but realized once you change any dependency to a class, the effects ripple through all the code if you want to use inheritance. was thinking we might want to take an hour and pair program to attempt a conversion!